$OpenBSD: patch-Source_WebKit2_UIProcess_StatisticsRequest_cpp,v 1.1 2013/10/06 14:42:39 landry Exp $
Taken from Source/WebKit2/Platform/CoreIPC/Connection.cpp
Needed on sparc64
--- Source/WebKit2/UIProcess/StatisticsRequest.cpp.orig	Thu Oct  3 21:41:28 2013
+++ Source/WebKit2/UIProcess/StatisticsRequest.cpp	Thu Oct  3 21:43:20 2013
@@ -47,7 +47,7 @@ uint64_t StatisticsRequest::addOutstandingRequest()
 {
     static int64_t uniqueRequestID;
 
-    uint64_t requestID = atomicIncrement(&uniqueRequestID);
+    uint64_t requestID = atomicIncrement(reinterpret_cast<int64_t volatile*>(&uniqueRequestID));
     m_outstandingRequests.add(requestID);
     return requestID;
 }
